composer.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "wordpress/twentynineteen",
+  "description": "A new Gutenberg-ready theme.",
+  "type": "package",
+  "license": "GPL v2 or later",
+  "keywords": [
+		"WordPress",
+		"Themes"
+	],
+  "authors": [
+		{
+			"name": "Contributors",
+			"homepage": "https://github.com/WordPress/twentynineteen/graphs/contributors"
+		}
+	],
+  "require": {
+      "wptrt/wpthemereview": "^0.1.0"
+  },
+  "require-dev": {
+      "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0"
+  },
+  "scripts": {
+		"phpcs-i": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs -i",
+		"check-cs": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs",
+		"fix-cs": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf"
```

Comment:
Can this use `format` as the script name please, this matches cores usage.

https://core.trac.wordpress.org/browser/trunk/composer.json#L18